wemake_python_styleguide/violations/complexity.py

```diff
@@ -1249,3 +1251,23 @@ class TooManyRaisesViolation(ASTViolation):
 
     error_template = 'Found too many raises in a function: {0}'
     code = 238
+
+
+@final
+class TooLongReturnTupleViolation(ASTViolation):
```

Comment:
Let's rename the existing `Yield` violation. I think that there's no need to create a new one.

Reply:
Okay.

Are there any docs in which cases changing violations or some parts of public API is ok?

Reply:
All done. I am not sure about naming, through.

Reply:
> Are there any docs in which cases changing violations or some parts of public API is ok?

We don't consider our code to be a part of our API, because no one in their sane minds would use it. This is a CLI tool, so only UX changes are important.

Reply:
I just am confused by the comment lines in `constants.py`. There are separation between constants public for "end-user" and internal which are not documented. So I do not get, what these lines are talking about if it is not about API. 

https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/constants.py#L25